projects/1.18/assets/alexs-cloud-storage/cloudstorage/lang/zh_cn.json

```
@@ -0,0 +1,112 @@
+  "cloudstorage.container.balloon_stand.color": "颜色：%s",
+  "cloudstorage.container.balloon_stand.slots": "%s / %s 个槽位",
+  "cloudstorage.container.balloon_stand.private_slots": "云：",
+  "cloudstorage.container.balloon_stand.public_slots": "带静电的云：",
+  "message.cloudstorage.no_sky_access": "云箱必须放在高空之下",
+  "message.cloudstorage.no_balloon": "这个云箱没有指定的气球",
+  "message.cloudstorage.no_inventory_space": "这种气球颜色没有对应的云背包",
+  "message.cloudstorage.no_static": "云箱不能挂上带静电的气球",
+  "death.attack.sneak_balloon_attack": "%s 被一只鬼鬼祟祟的气球怪杀死了",
+  "advancements.cloudstorage.root.title": "云储存",
+  "advancements.cloudstorage.root.desc": "天空即为极限",
+  "advancements.cloudstorage.badloon.title": "放气",
+  "advancements.cloudstorage.badloon.desc": "在一只气球怪击破你之前击破它",
+  "advancements.cloudstorage.bloviator.title": "§mSky of Beginning§r 烦人的天空",
+  "advancements.cloudstorage.bloviator.desc": "杀死Bloviator以免你坠入死亡",
+  "advancements.cloudstorage.balloon.title": "不可击破",
+  "advancements.cloudstorage.balloon.desc": "用气球碎块和线来制造一个气球",
+  "advancements.cloudstorage.angry_cloud_in_a_bottle.title": "永远封印",
+  "advancements.cloudstorage.angry_cloud_in_a_bottle.desc": "把一只微小的Bloviator抓进瓶子里",
+  "advancements.cloudstorage.happy_cloud_in_a_bottle.title": "如果你给云朵一片饼干",
+  "advancements.cloudstorage.happy_cloud_in_a_bottle.desc": "通过把一团云朵和曲奇一起结合来让云朵开心",
+  "advancements.cloudstorage.cloud_chest.title": "天堂里还有一些空间",
+  "advancements.cloudstorage.cloud_chest.desc": "用八个云块和一个箱子制造一个云箱",
+  "advancements.cloudstorage.static_cloud.title": "带正电",
+  "advancements.cloudstorage.static_cloud.desc": "通过在风暴之中杀死Bloviator来获得带静电的云块",
+  "advancements.cloudstorage.static_balloon.title": "汗毛倒竖",
+  "advancements.cloudstorage.static_balloon.desc": "通过对气球使用带静电的云块来给它充上静电",
+  "advancements.cloudstorage.static_cloud_chest.title": "公共储存单元",
+  "advancements.cloudstorage.static_cloud_chest.desc": "用八个带静电的云块和一个箱子制造一个带静电的云箱",
+  "advancements.cloudstorage.upload.title": "向上、加载并上传",
+  "advancements.cloudstorage.upload.desc": "通过把气球系在箱子上来提起箱子，以便为对应气球颜色的云背包添加存储空间",
+  "advancements.cloudstorage.balloon_buddy.title": "真正的朋友是陪你直到最后的最后的朋友",
+  "advancements.cloudstorage.balloon_buddy.desc": "制造一只气球伙伴",
+  "advancements.cloudstorage.99_balloons.title": "99只气球",
+  "advancements.cloudstorage.99_balloons.desc": "一次性在你的背包中拥有99只气球",
+  "advancements.cloudstorage.balloon_stand.title": "全国免费气球日",
+  "advancements.cloudstorage.balloon_stand.desc": "制作一个气球摊以便让复制气球颜色更加简便",
+  "advancements.cloudstorage.propeller_hat.title": "气球男孩",
+  "advancements.cloudstorage.propeller_hat.desc": "获得一顶螺旋桨帽，它会取决于天气地旋转得更快",
+  "advancements.cloudstorage.cotton_candy.title": "甜蜜之吻",
+  "advancements.cloudstorage.cotton_candy.desc": "通过糖和一块云块获得棉花糖",
+  "advancements.cloudstorage.string_on_balloon.title": "再见了，小伙子们！",
+  "advancements.cloudstorage.string_on_balloon.desc": "通过对气球绳使用线来使它变得更长",
+  "advancements.cloudstorage.all_balloon_buddies.title": "现在我们都是朋友了",
+  "advancements.cloudstorage.all_balloon_buddies.desc": "收集所有的8种个性的气球伙伴",
```

```suggestion
  "advancements.cloudstorage.all_balloon_buddies.desc": "收集全部8种个性的气球伙伴",
  ```
  原文：**Collect all 8 balloon buddy personalities**
  | 严重程度         | 备注 |
  |--------------|----|
  | ⚪ 轻微 / 不阻止合并但需三思  |    |

---

ok